hook.php

```diff
@@ -86,9 +86,30 @@ function plugin_escalade_install() {
       ) ENGINE=InnoDB DEFAULT CHARSET={$default_charset} COLLATE={$default_collation} ROW_FORMAT=DYNAMIC;";
       $DB->query($query);
 
-      $query = "INSERT INTO glpi_plugin_escalade_configs
-      VALUES (NULL, 1, 1, 1, 1, 1, 0, 0, 1, 1, 0, 0, 0, 1, 1, 0, 1, 1, 0, 1, 0, '".Ticket::WAITING."',0)";
-      $DB->query($query);
+      $DB->insert('glpi_plugin_escalade_configs', [
+         'remove_group' => 1,
+         'show_history' => 1,
+         'task_history' => 1,
+         'remove_tech' => 1,
+         'solve_return_group' => 1,
+         'reassign_group_from_cat' => 0,
+         'reassign_tech_from_cat' => 0,
+         'cloneandlink_ticket' => 1,
+         'close_linkedtickets' => 1,
+         'use_assign_user_group' => 0,
+         'use_assign_user_group_creation' => 0,
+         'use_assign_user_group_modification' => 0,
+         'remove_delete_requester_user_btn' => 1,
+         'remove_delete_watcher_user_btn' => 1,
+         'remove_delete_assign_user_btn' => 0,
+         'remove_delete_requester_group_btn' => 1,
+         'remove_delete_watcher_group_btn' => 1,
+         'remove_delete_assign_group_btn' => 0,
+         'remove_delete_assign_supplier_btn' => 1,
+         'use_filter_assign_group' => 1,
```

Comment:
It was 0 on previous code.
```suggestion
         'use_filter_assign_group' => 0,
```